src/tiles/selection-tile.component.ts

```diff
@@ -19,18 +19,20 @@ import { I18n } from "carbon-components-angular/i18n";
 			[for]="id"
 			[ngClass]="{
 				'bx--tile--is-selected' : selected,
-				'bx--tile--light': theme === 'light'
+				'bx--tile--light': theme === 'light',
+				'bx--tile--disabled' : disabled
```

Comment:
Please update **line 18** (tabindex), when disabled, it should be set to `null` removing the attribute to match React. Otherwise, users will still be able to `focus` the `disabled` tile.

![image](https://user-images.githubusercontent.com/38994122/218886157-401c7270-af8d-453d-8590-1dca6ef317b9.png)

Reply:
Updated